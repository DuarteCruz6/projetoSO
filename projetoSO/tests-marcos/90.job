WRITE [(t7,37)(t5,16)(b6,1)(h9,7)(u9,11)]
READ [b6]
DELETE [b6,t7,u9,h9]
BACKUP
WRITE [(w7,79)(h5,74)(f7,86)(q8,42)]
READ [t5]
WRITE [(z5,20)(c3,4)(r8,4)(p1,7)(l1,90)]
WRITE [(r7,90)(w8,18)(m7,29)(m7,55)]
READ [l1]
WRITE [(s4,26)(l9,76)]
READ [p1]
WRITE [(g3,24)(i4,88)(k1,19)]
READ [r8]
WRITE [(c6,41)(p3,100)]
WRITE [(t3,15)(d3,74)(m8,65)]
WRITE [(w4,16)(l7,32)(h7,64)(p8,58)(b6,3)]
DELETE [l9,p1]
SHOW
WRITE [(k9,37)(v1,65)]
SHOW
WRITE [(c7,5)(g0,30)(b7,32)(f8,49)(c7,59)]
WRITE [(a3,97)(z4,74)]
BACKUP
WRITE [(q4,22)(z2,26)(a7,85)(q9,25)(x5,91)]
WRITE [(n5,53)(u9,77)(s3,13)(x8,52)]
WRITE [(x9,19)(w5,13)(u4,33)]
SHOW
SHOW
WRITE [(r0,45)(n4,49)(n9,66)(h1,75)(c9,68)]
READ [k1]
WRITE [(h3,53)(l3,1)]
WRITE [(h6,11)(l0,5)(a1,30)(s0,53)]
WRITE [(q5,85)(q7,33)(d5,75)(k2,68)(x0,72)]
WRITE [(g2,73)]
WRITE [(e6,27)(a6,68)(u7,10)(m2,45)(i6,87)]
WRITE [(d1,36)(g8,78)(c2,66)(i7,1)]
WRITE [(n0,94)(m1,96)(y5,89)]
READ [q4]
DELETE [h7,h1,m1]
BACKUP
WRITE [(y4,87)(u5,91)(m0,35)(c1,44)(x3,17)]
READ [i6]
WRITE [(p7,9)(q3,95)(a5,93)(u6,46)]
WRITE [(j7,97)(e9,10)(j0,64)(z6,62)]
WRITE [(h2,6)(s2,69)]
WRITE [(n3,25)(g4,55)]
WRITE [(e1,70)(x6,59)(p2,64)(y2,22)]
WRITE [(t8,82)(q2,4)(p9,61)(l5,58)(a2,69)]
READ [t5]
WRITE [(o4,76)(p1,6)]
WRITE [(e2,94)]
SHOW
