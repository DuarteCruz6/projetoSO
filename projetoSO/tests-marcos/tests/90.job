WAIT 2000
WRITE [(d5,23)(m1,87)]
WRITE [(z9,85)(e2,90)(q1,6)(b6,71)(q2,19)]
DELETE [d5,q1]
DELETE [b6,z9,e2,m1,q2]
SHOW
WRITE [(n1,72)(b4,46)]
WRITE [(w4,68)(y8,45)]
WRITE [(e9,8)(a4,30)(b0,30)]
READ [n1]
WRITE [(m5,69)(c4,75)(i3,20)]
READ [b4]
WRITE [(l9,45)(e8,64)]
READ [a4]
WRITE [(n6,43)(k5,98)(q1,62)(d0,45)(h7,59)]
WAIT 3000
DELETE [d0]
WAIT 2000
DELETE [m5,l9,w4,n1]
WRITE [(t9,80)(h9,9)]
SHOW
DELETE [y8,b4]
DELETE [h9,q1,e9]
WRITE [(u9,77)(t8,7)(z8,94)]
WRITE [(a5,33)(o8,79)(z7,55)]
BACKUP
WRITE [(f8,92)(l8,92)(k8,89)(p4,38)]
BACKUP
READ [a5]
WRITE [(z4,83)]
DELETE [b0,o8]
WRITE [(r8,80)(m3,32)(x6,48)]
WRITE [(z5,51)(i8,76)(m4,53)(x9,66)]
BACKUP
WRITE [(c8,46)]
DELETE [a5,z7,h7]
WRITE [(s1,80)(c7,63)(g6,46)(u0,37)(c2,60)]
DELETE [t8,a4,k8,i3]
WRITE [(w3,77)(x3,88)]
WAIT 3000
DELETE [c8]
WAIT 3000
WRITE [(t4,92)(t1,29)(p9,21)]
WRITE [(v1,44)(q8,7)(k6,68)(u3,11)]
DELETE [w3,x6,r8,s1,f8]
WRITE [(n0,30)(l4,12)(m2,64)]
READ [e8]
DELETE [u9,u3,p4,l4,m4]
WRITE [(j4,62)(u4,37)(a4,13)(h3,9)(b6,88)]
DELETE [u4,z4,u0,x3,b6]
READ [k5]
READ [v1]
WAIT 3000
WRITE [(d9,61)]
WAIT 3000
DELETE [n0,m3,c2,v1]
DELETE [t1,c7]
WRITE [(r1,92)(c6,62)]
WAIT 2000
READ [r1]
WAIT 1000
WRITE [(d6,40)(j3,56)(m4,4)(y2,32)(g8,43)]
WRITE [(c1,48)]
WRITE [(z1,66)(z4,5)(y3,1)(q0,14)(e7,45)]
BACKUP
READ [k6]
WRITE [(d0,75)(y9,74)(p1,28)(o6,3)]
