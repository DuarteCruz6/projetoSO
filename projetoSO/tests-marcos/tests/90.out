[(n1,72)]
[(b4,46)]
[(a4,30)]
(a4, 30)
(b0, 30)
(b4, 46)
(c4, 75)
(e8, 64)
(e9, 8)
(h9, 9)
(h7, 59)
(i3, 20)
(k5, 98)
(n6, 43)
(q1, 62)
(t9, 80)
(y8, 45)
[(a5,33)]
[(e8,64)]
[(k5,98)]
[(v1,44)]
[(r1,92)]
[(k6,68)]
